ckanext/datastore/backend/postgres.py

```diff
@@ -488,6 +499,16 @@ def _where_clauses(
     return clauses
 
 
+def _prepare_where_operator_and_value(value: dict[str, Any]) -> tuple[str, Any]:
+    try:
+        key = next(iter(value))
+        val = value[key]
+    except (StopIteration, KeyError):
+        return '=', value
```

Comment:
we can ignore the multiple-value dict case too with:
```suggestion
    try:
        [(key, val)] = value.items()
    except ValueError:
        return '=', value
```